### PR TITLE
HAAM-Radio-October-event

### PR DESCRIPTION
**Pull request by TinaSaupe**
Create HAAM-Radio event for October 2024

### CODE REVIEW

**Review by TinaSaupe — COMMENTED** [review]
Yes. should be.

**Comment by bamorim-bio** [comment]
@TinaSaupe check if changes are ok :)

File: _posts/2024-09-19-event.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "HAAM-Radio #3 - October 2024: Population demography and social organisations in the past of Southeast Asia and Oceania"
+categories: Events
+tags: HAAM-Radio
+---
+
+The next inaugural HAAM-Radio Webinar will be taking place online on the 2nd of October, 2024 at 7.00 - 9.00 EST/ 13.00 - 15.00 CET/ 23.30 - 22.30 ACST! This webinar will focus on population demography and social organisations in the past of Southeast Asia and Oceania.
```
Comment (TinaSaupe) [inline _posts/2024-09-19-event.md:8]: Could you change the ! to .

**Review by TinaSaupe — COMMENTED** [review]
Fine :)

**Pull request merged**